wof/examples/flask/odm2/timeseries/odm2_timeseries_dao.py

```diff
@@ -128,18 +128,33 @@ def get_variables_from_results(self, var_codes=None):
 
         r_t_Arr = []
         if l_var_codes is None:
-            r_t = self.db_session.query(odm2_models.TimeSeriesResultValues). \
-                join(odm2_models.TimeSeriesResults). \
-                group_by(odm2_models.TimeSeriesResults.VariableID).all()
+            if self.engine.name is 'postgresql':
+                r_t = self.db_session.query(odm2_models.TimeSeriesResultValues). \
+                    join(odm2_models.TimeSeriesResults). \
+                    distinct(odm2_models.TimeSeriesResults.VariableID).all()
+            else:
+                r_t = self.db_session.query(odm2_models.TimeSeriesResultValues). \
+                    join(odm2_models.TimeSeriesResults). \
+                    group_by(odm2_models.TimeSeriesResults.VariableID).all()
```

Comment:
For reference: We hope this postgresql-specific separation of SQLAlchemy queries can be removed in a future update, so we can maintain common SQLAlchemy (or odm2api-based!) querying code w/o such "hard wiring"